.circleci/config.yml

```diff
@@ -275,7 +275,7 @@ jobs:
           command: rustc --version; cargo --version; rustup --version; rustup target list --installed
       - restore_cache:
           keys:
-            - cargocache-v2-bindings:1.54.0-{{ checksum "~/project/Cargo.lock" }}
```

Comment:
this could be another pr, right?
or bad git diff?

Reply:
This is lack of rebase from main I think. Will fix.

Reply:
No, this was a change I did in passing, after changing this file because of some CI error.